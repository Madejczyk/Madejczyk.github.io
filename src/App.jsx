export const App = ({ callback }) => {
    return (
        <>
            <h1>Hello World</h1>
        </>
    );
}