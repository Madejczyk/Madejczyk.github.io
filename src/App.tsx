import * as React from 'react';

const App = () => {
    return (
        <div>
            <h1>I am also waiting for this...</h1>
        </div>
    );
}
export default App;