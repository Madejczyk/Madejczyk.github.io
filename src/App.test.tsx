import * as React from 'react';
import {render} from '@testing-library/react'
import '@testing-library/jest-dom'
import App from './App'

test('renders a message', () => {
  const {container, getByText} = render(<App />)
  expect(getByText('I am also waiting for this...')).toBeInTheDocument()
  expect(container.firstChild).toMatchInlineSnapshot(`
  <div>
    <h1>
      I am also waiting for this...
    </h1>
  </div>
  `)
})