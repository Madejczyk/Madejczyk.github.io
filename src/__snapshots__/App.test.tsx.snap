// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders a message 1`] = `
<div>
  <div>
    I am 
  </div>
  <div>
    <span>
      also waiting for this...
    </span>
  </div>
</div>
`;
